### DISCUSSION TITLE
Add parameters to semanticPrefixFixDepsChoreOthers

### DISCUSSION THREAD

**Discussion by glensc** [renovatebot/renovate#22114] [Suggest an Idea]
### Type of discussion.

I'm proposing an idea

### Tell us more.

Existing rule:
- https://docs.renovatebot.com/presets-default/#semanticprefixfixdepschoreothers

Our commits policy has the semver category uppercased, so would be useful if that macro takes Fix and Chore as parameters

```
": semanticPrefixFixDepsChoreOthers(Fix, Chore)"
```

**Comment by glensc** [comment]
I copied. that contents manually to renovate.json5, but the prefixes are still with lowercase letters. why?


```json
{
      "matchPackagePatterns": [
        "*"
      ],
      "semanticCommitType": "Chore"
    },
    {
      "matchDepTypes": [
        "dependencies",
        "require"
      ],
      "semanticCommitType": "Fix"
    },
    {
      "matchDatasources": [
        "maven"
      ],
      "matchDepTypes": [
        "compile",
        "provided",
        "runtime",
        "system",
        "import",
        "parent"
      ],
      "semanticCommitType": "Fix"
    }
```

**Reply by rarkins** [reply]
See #12062

**Reply by glensc** [reply]
or now this with a config:
- https://github.com/renovatebot/renovate/pull/20930
